Examples/Tests/photon_pusher/analysis_photon_pusher.py

```diff
@@ -152,7 +152,7 @@ def generate():
             f.write("{}.single_particle_u = {} {} {}\n".
                 format(name, velx, vely, velz))
             f.write("{}.single_particle_weight = 1.0\n".format(name))
-            f.write("\n".format(name))
+            f.write("\n".format())
```

Comment:
This should be just `f.write("\n")`

Reply:
Thx! Added to #5124, will rebase out here later.